Sources/Charcoal/Filters/List/ListFilterViewController.swift

```diff
@@ -159,3 +224,18 @@ extension ListFilterViewController: UITableViewDelegate {
         }
     }
 }
+
+// MARK: - UISearchBarDelegate
+
+extension ListFilterViewController: UISearchBarDelegate {
+    public func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
+        if searchText.isEmpty {
+            scopedSubfilters = filter.subfilters
+        } else {
+            let searchTextLowercased = searchText.lowercased()
+            scopedSubfilters = filter.subfilters.filter { $0.title.lowercased().contains(searchTextLowercased) }
+        }
```

Comment:
Just a little tip or "nice-to-have" thing we discovered when adding a similar feature in `ad-insertion-ios`:

It could be a good idea to sort the matches by prefix matching here as well, to avoid "less" important matches occurring before the exact matches. I.e. if you search for "Land Rover" and type "La" you would expect "Land Rover" and "Lamborghini" to occur before "Cadillac" 🙃

```
scopedSubfilters = filter.subfilters
    .filter { $0.title.lowercased().contains(searchTextLowercased) }
    .sorted {
        let first = $0.title.lowercased().hasPrefix(query) ? 0 : 1
        let second = $1.title.lowercased().hasPrefix(query) ? 0 : 1
        return first < second
    }
```

Reply:
That sounds pretty clever! I'll try it out 🌟

Reply:
I think this makes much more sense 😍
![Screenshot 2020-05-28 at 14 59 06](https://user-images.githubusercontent.com/1901556/83144590-dc324300-a0f3-11ea-9968-1b093e4b9786.png)

Reply:
Nice! :D I think so too :)